src/test/scala/chiselTests/BoringUtilsTapSpec.scala

```diff
@@ -573,6 +677,93 @@ class BoringUtilsTapSpec extends ChiselFlatSpec with ChiselRunners with Utils wi
     val verilog = circt.stage.ChiselStage.emitSystemVerilog(new Top(Definition(new Foo)))
   }
 
+  it should "work to rwTap an Instance[..]'s port" in {
+    import chisel3.experimental.hierarchy._
+    class UnitTestHarness extends Module {
+      val dut = Instantiate(new Dut)
+      probe.force(dut.widgetProbes.head, 0xffff.U)
+    }
+
+    @instantiable
+    class Widget extends Module {
+      @public val in = IO(Input(UInt(32.W)))
+      @public val out = IO(Output(UInt(32.W)))
+      out := ~in
+    }
+
+    @instantiable
+    class Dut extends Module {
+      val widgets: Seq[Instance[Widget]] = Seq.tabulate(1) { i =>
+        val widget = Instantiate(new Widget)
+        widget.in := i.U
+        widget
+      }
+      @public val widgetProbes = widgets.map { widget =>
+        val widgetProbe = IO(probe.RWProbe(UInt(32.W)))
+        val define = BoringUtils.rwTap(widget.out)
+        probe.define(widgetProbe, define)
+        widgetProbe
+      }
+    }
+    matchesAndOmits(circt.stage.ChiselStage.emitCHIRRTL(new UnitTestHarness))(
+      "module Widget :",
+      "input clock : Clock",
+      "input reset : Reset",
+      "input in : UInt<32>",
+      "output out : UInt<32>",
+      "node _out_T = not(in)",
+      "connect out, _out_T",
+      "module Dut :",
+      "define widgetProbes_0 = rwprobe(widgets_0.out)",
+      "public module UnitTestHarness :",
+      "force(clock, _T, dut.widgetProbes_0, UInt<32>(0hffff))"
+    )()
+  }
+
+  it should "work to tap an Instance[..]'s port" in {
```

Comment:
this looks good, wondering if you wouldn't want a similar test to the one above where if you `tap` to a Probe port it should work. But not blocking my review on that